.github/workflows/auto-badge.yml

```
@@ -0,0 +1,34 @@
+on:
+  issues:
+    types: [opened, edited, reopened]
+
+jobs:
+  add-badge:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Add Activity Badge
+        uses: actions/github-script@v6
```

actionlint (1.7.7)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)